promptsource/templates/quac/templates.yaml

```diff
@@ -0,0 +1,90 @@
+dataset: quac
+templates:
+  12c9d007-991c-49ed-82e2-13a7d3147881: !Template
+    answer_choices: null
+    answer_choices_key: null
+    id: 12c9d007-991c-49ed-82e2-13a7d3147881
+    jinja: "Given the dialogue : \n\nStudent: {{questions[0]}}\n\nTeacher: {{answers.texts[0]\
```

Comment:
Can you make it more clear in the prompt that that this only is given part of the information that it needs to get the right answer.